.github/workflows/ci.yaml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
@richardmarshall Should actually be `main`?

Reply:
Yes! Thank you for catching that.

Reply:
fixed